### ISSUE TITLE
Rendering convex hull

### ISSUE THREAD

**Issue by btickell** [openai/mujoco-py#461]
The simulate executable for mujoco allows for rendering the convex hull version of the scene which is helpful for analyzing the contact model of certain geoms.

Is it possible to do the same thing in `sim.render()` or a similar python setting?

**Comment by priyankamandikal** [comment]
Is there any update on this? How do we visualize the convex hull in the renderer?

**Comment by priyankamandikal** [comment]
btickell, were you able to figure out how to render the convex meshes?

**Comment by btickell** [comment]
I ended up converting my meshes into convex meshes.

**Comment by priyankamandikal** [comment]
Using the HACD library?

**Comment by btickell** [comment]
Yes. This seems to be the standard approach from people I've talked to. I think rendering the hull would still be a useful feature.

**Comment by priyankamandikal** [comment]
After some fiddling around with the code, I got it working with a small fix:
sim = MjSim(model)
viewer = MjViewer(sim)
viewer.vopt.flags[0] = 1

Hope this helps.

**Comment by right-chan** [comment]
It was late, but I did it like this.


```python
import mujoco_py
from mujoco_py import load_model_from_path, MjSim, MjViewer
import os

mj_path = mujoco_py.utils.discover_mujoco()
xml_path = os.path.join(mj_path, 'model', 'humanoid.xml')

model = load_model_from_path(xml_path)
sim = MjSim(model)
viewer = MjViewer(sim)

viewer.vopt.flags[0] = True
viewer.render()
```
I found it [here](https://mujoco.readthedocs.io/en/latest/APIreference.html?highlight=mjVIS_CONVEXHULL#mjtvisflag).
